python/cudf/dataframe/string.py

```diff
@@ -458,6 +465,14 @@ def __len__(self):
     def dtype(self):
         return self._dtype
 
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, name):
+        self._name = name
```

Comment:
You can remove this and the property and have it inherit it from the base `Column` class

Reply:
Done